### DISCUSSION TITLE
Kaspersky Endpoint Security Trojan Horse Signature on Windows 10 21H2

### DISCUSSION THREAD

**Discussion by jkellywilkerson** [actix/actix-web#2578] [Q&A]
Greetings All:

Doing the following on Windows 10 with Kaspersky Endpoint Security produces Trojan Horse flagged files in the /target folder:

Cargo.toml Dependencies:

[dependencies]
actix-web = "3"

main.rs Code:

use actix_web::{get, web, App, HttpServer, Responder};

#[get("/{id}/{name}/index.html")]
async fn index(web::Path((id, name)): web::Path<(u32, String)>) -> impl Responder {
    format!("Hello {}! id:{}", name, id)
}

#[actix_web::main]
async fn main() -> std::io::Result<()> {
    HttpServer::new(|| App::new().service(index))
        .bind("127.0.0.1:8080")?
        .run()
        .await
}

Any ideas or insight as to how I can use the actix-web dependency? Unable to build even a minimal web server or web service due to all the /target files being locked and/or quarantined.

Thanks.

**Comment by robjtede** [comment]
I would imagine there's a way to tell an AV system to allow specific programs if they're falsely flagged as problematic, as is the case here. I can't help much more than that.
